packages/quantic/force-app/main/default/lwc/quanticSort/e2e/pageObject.ts

```diff
@@ -0,0 +1,73 @@
+import type {Locator, Page, Request} from '@playwright/test';
+import {isUaSearchEvent} from '../../../../../../playwright/utils/requests';
+
+export class SortObject {
+  constructor(public page: Page) {
+    this.page = page;
+  }
+
+  get sortDropDown(): Locator {
+    return this.page.getByRole('combobox', {name: 'Sort by'});
+  }
+
+  get sortPreviewButton(): Locator {
+    return this.page.getByRole('button', {name: 'Preview'});
+  }
+
+  sortButton(buttonName: string): Locator {
+    return this.page.getByRole('option', {name: buttonName});
+  }
+
+  async clickSortDropDown(): Promise<void> {
+    await this.sortDropDown.click();
+  }
+
+  async focusSortDropDown(): Promise<void> {
+    await this.sortPreviewButton.click();
+    await this.page.keyboard.press('Tab');
+  }
+
+  async clickSortButton(buttonName: string): Promise<void> {
+    await this.sortButton(buttonName).click();
+  }
+
+  async openSortDropdownUsingEnter(useEnter = true): Promise<void> {
+    if (useEnter) {
+      await this.page.keyboard.press('Enter');
+    } else {
+      await this.page.keyboard.press('Space');
+    }
+  }
+
+  async selectSortOptionUsingArrow(): Promise<void> {
+    await this.sortButton('Oldest').isVisible();
+    await this.page.waitForTimeout(500);
+    await this.page.keyboard.press('ArrowDown');
+    await this.page.keyboard.press('Enter');
+  }
```

Comment:
This one is weird for me, why you rely on 'Oldest' specifically, and also why do you have to wait for 500ms?

Reply:
After seeing the test, it shouldn't be in the `pageObject` just in your test, put this:

```
await this.page.keyboard.press('ArrowDown');
await this.page.keyboard.press('Enter');
```

Reply:
the `wait for 500ms` is to make the test for Insight use case more stable, otherwise the next keyboard action sometimes will fail.

Reply:
Ah yes I remember that, it's true, I think you should put indeed a small delay in between all keyboard press if I remember correctly.